src/components/Organizations/Details/Transfers.tsx

```diff
@@ -0,0 +1,187 @@
+  }
+
+  return (
+    <Tooltip label={tooltip}>
+      <span>
+        <Icon as={icon} color={color} {...rest} />
+      </span>
+    </Tooltip>
+  )
+}
+
+interface AccountTransfersProps {
+  txCount: number | undefined
+  org: AccountData
+}
+
+const AccountTransfers = (txProps: AccountTransfersProps) => {
+  return (
+    <PaginationProvider totalPages={Math.ceil(txProps.txCount ?? 0 / PaginationItemsPerPage)}>
+      <AccountTransfersTable {...txProps} />
+    </PaginationProvider>
+  )
+}
+
+const AccountTransfersTable = ({ txCount, org }: AccountTransfersProps) => {
+  const { page } = usePagination()
+  const { formatDistance } = useDateFns()
+
+  const { data, isLoading, isError, error } = useAccountTransfers({
+    address: org.address,
+    page: Number(page) - 1 || 0,
+    options: {
+      enabled: !!txCount && txCount > 0,
+      retry: retryUnlessNotFound,
+    },
+  })
+
+  if (txCount && !(txCount > 0)) {
+    return (
+      <Text>
+        <Trans i18nKey={'organization.transfers.no_transfers'}>No transfers yet!</Trans>
+      </Text>
+    )
+  }
+
+  if (isError) {
+    return <LoadingError error={error} />
+  }
+
+  if (!txCount || isLoading) {
+    return <LoadingCards />
+  }
+
+  let mergedTransfers: IAccountTransfer[] = []
+  if (data) {
+    mergedTransfers = [...data.transfers.received, ...data.transfers.sent]
+    mergedTransfers.sort((a, b) => new Date(a.timestamp).getTime() - new Date(b.timestamp).getTime()).reverse()
+  }
+
+  return (
+    <>
+      <Box overflow='auto' w='auto'>
+        <TableContainer>
+          <Table>
+            <Thead>
+              <Tr>
+                <Th>
+                  <Trans i18nKey={'organization.transfers.tx_hash'}>Tx Hash</Trans>
+                </Th>
+                <Th>
+                  <Trans i18nKey={'organization.transfers.block'}>Block</Trans>
+                </Th>
+                <Th>
+                  <Trans i18nKey={'organization.transfers.from_to'}>From/To</Trans>
+                </Th>
+                <Th>
+                  <Trans i18nKey={'organization.transfers.amount'}>Amount</Trans>
+                </Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              {mergedTransfers.map((transfer, i) => {
+                const isIncoming = transfer.to === org.address
+                let fromToAddress = isIncoming ? transfer.from : transfer.to
+                return (
+                  <Tr key={i}>
+                    <Td>
+                      <Flex direction={'column'} align={'start'} gap={3}>
+                        <ReducedTextAndCopy
+                          breakPoint={{ base: true }}
+                          color={'textAccent1'}
+                          toCopy={transfer.txHash}
+                          fontWeight={'normal'}
+                          fontSize={'sm'}
+                          // todo: implement go to tx by its hash
```

Comment:
This todo can be fixed when https://github.com/vocdoni/explorer/pull/58 is merged